windows/msys/run_tests.sh

```diff
@@ -9,7 +9,7 @@ CFLAGS_EXTRA=$2         # Extra C flags
 
 LOG=run_tests.log
 
-CFLAGS="-g -Igdtest/ -I. -I../src/ -D_WIN32 $CFLAGS_EXTRA"
+CFLAGS="-g -Igdtest/ -I. -I../src/ -D_WIN32 -DHAVE_SYS_STAT_H $CFLAGS_EXTRA"
```

Comment:
`-DHAVE_SYS_STAT_H` is necessary for tests/gdtest/gdtest.c to include "sysy/stat.h", which defines the `struct stat` used in function _clean_dir().